src/sql/workbench/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -182,6 +185,17 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		}
 	}
 
+	private toggleUserSelect(userSelect: boolean): void {
```

Comment:
Dumb question: why do we need to toggle this? Why can't it be on all the time?

Reply:
If it is on all the time, then all of them will be selected. This is the way I came up. Maybe there are better way to do it.